gateway-shell/src/test/java/org/apache/knox/gateway/shell/table/KnoxShellTableTest.java

```diff
@@ -365,6 +373,67 @@ public Object answer() {
     verify(connection, statement, resultSet, metadata);
   }
 
+  @Test
+  public void testJDBCBuilderUsingConnectionString() throws Exception {
+    final String connectionUrl = "testConnectionUrl";
+    ResultSetMetaData metadata = createMock(ResultSetMetaData.class);
+    try (Connection connection = EasyMock.createNiceMock(Connection.class);Statement statement = createMock(Statement.class);ResultSet resultSet = createMock(ResultSet.class)){
+      PowerMock.mockStatic(DriverManager.class);
```

Comment:
Can you add a comment to this line about how need PowerMock since EasyMock can't mock statics?